test/semantics/io03.f90

```diff
@@ -0,0 +1,146 @@
+! Copyright (c) 2019, NVIDIA CORPORATION.  All rights reserved.
+!
+! Licensed under the Apache License, Version 2.0 (the "License");
+! you may not use this file except in compliance with the License.
+! You may obtain a copy of the License at
+!
+!     http://www.apache.org/licenses/LICENSE-2.0
+!
+! Unless required by applicable law or agreed to in writing, software
+! distributed under the License is distributed on an "AS IS" BASIS,
+! WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+! See the License for the specific language governing permissions and
+! limitations under the License.
+
+  character(kind=1,len=50) internal_file
+  character(kind=2,len=50) internal_file2
```

Comment:
"kind=4", viz. Unicode, is the other one that we want to support.

Reply:
I've added a kind=4 declaration to io03.f90